### ISSUE TITLE
Build fails with Poppler 22.04: no matching constructor for initialization of '::PDFDoc'

### ISSUE THREAD

**Issue by yurikoles** [jwilk/pdf2djvu#153]
```
  pdf-backend.cc:133:3: error: no matching constructor for initialization of '::PDFDoc'
  : ::PDFDoc(new pdf::String(file_name.c_str()), nullptr, nullptr)
    ^        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  /opt/homebrew/Cellar/poppler/22.04.0/include/poppler/PDFDoc.h:132:14: note: candidate constructor not viable: no known conversion from 'pdf::String *' (aka 'GooString *') to 'std::unique_ptr<GooString>' for 1st argument
      explicit PDFDoc(std::unique_ptr<GooString> &&fileNameA, const std::optional<GooString> &ownerPassword = {}, const std::optional<GooString> &userPassword = {}, void *guiDataA = nullptr,
               ^
  /opt/homebrew/Cellar/poppler/22.04.0/include/poppler/PDFDoc.h:139:14: note: candidate constructor not viable: no known conversion from 'pdf::String *' (aka 'GooString *') to 'BaseStream *' for 1st argument
      explicit PDFDoc(BaseStream *strA, const std::optional<GooString> &ownerPassword = {}, const std::optional<GooString> &userPassword = {}, void *guiDataA = nullptr, const std::function<void()> &xrefReconstructedCallback = {});
               ^
  /opt/homebrew/Cellar/poppler/22.04.0/include/poppler/PDFDoc.h:142:5: note: candidate constructor not viable: requires 1 argument, but 3 were provided
      PDFDoc(const PDFDoc &) = delete;
      ^
  /opt/homebrew/Cellar/poppler/22.04.0/include/poppler/PDFDoc.h:371:5: note: candidate constructor not viable: requires 0 arguments, but 3 were provided
      PDFDoc();
      ^
```

**Comment by jwilk** [comment]
I think that's already fixed in git (commits e170ad557d5f13da, 81b635e014ebd024, 956fedc7e0831126), but I haven't made a release yet.

**Comment by yurikoles** [comment]
Thank you and sorry for the noise. I [added](https://github.com/Homebrew/homebrew-core/pull/98905/commits/c4b7b7eab90a489bab0b5f86ebb1a688aa4a0478) those commits to homebrew.

**Comment by jwilk** [comment]
Let's keep the bug open until it's fixed in a release.

**Comment by jwilk** [comment]
Fixed in [0.9.19](https://github.com/jwilk/pdf2djvu/releases/tag/0.9.19).
